geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/accumulo/commands/ExplainCommand.scala

```diff
@@ -32,5 +34,7 @@ class ExplainCommand(parent: JCommander) extends CommandWithCatalog(parent) with
 
 object ExplainCommand {
   @Parameters(commandDescription = "Explain how a GeoMesa query will be executed")
-  class ExplainParameters extends RequiredCqlFilterParameters {}
+  class ExplainParameters extends GeoMesaConnectionParams
```

Comment:
Another fun trick is Parameters delegate! http://jcommander.org/#Parameter_delegates